Update Readme to add ZK migration instructions (#66)
PostHog/charts-clickhouse: charts/posthog/README.md

    <b>Digital Ocean</b>
  </summary>
<br />

### Requirements
**Required minimum resources to run Posthog are 4vcpu and 4G of memory for cluster nodes.**

> There is a [1-click option to deploy Posthog](https://marketplace.digitalocean.com/apps/posthog-1) on DigitalOcean. If you'd like to install the chart with `helm` directly continue reading.
>   
> ### `doctl` to install the 1-click stack directly:
> ```console
>   doctl kubernetes cluster create posthog-cluster --count=2 --size="s-2vcpu-4gb" --region=sfo3 --1-clicks=posthog
> ```

Reviewer: ```suggestion
### 1-click install:
There is a [1-click option to deploy Posthog](https://marketplace.digitalocean.com/apps/posthog-1) on DigitalOcean Marketplace UI or by using `doctl`

```console
  doctl kubernetes cluster create posthog-cluster --count=2 --size="s-2vcpu-4gb" --region=sfo3 --1-clicks=posthog
```
```

```

Run the following and `helm upgrade` again: `kubectl delete --namespace NAMESPACE sts posthog-posthog-redis-master`
  
### Upgrading from 2.x.x
  
3.0.0 changes the way ZK is run in the chart. ZK has been spun out and is now a cluster being used for Kafka and ClickHouse. An unfortunate side effect of that is that Kafka statefulset must be deleted. The reason for this is because Kafka records the cluster ID in ZooKeeper and when you swap it out it complains that the cluster ID has changed and refuses to start.
  
The error you will see from Kafka pod while upgrading:
```
  [2021-07-23 14:24:27,143] ERROR Fatal error during KafkaServer startup. Prepare to shutdown (kafka.server.KafkaServer)
kafka.common.InconsistentClusterIdException: The Cluster ID TYP8xsIWRFWkzSYmO_YWEA doesn't match stored clusterId Some(CizxEcefTou4Ehu65rmpuA) in meta.properties. The broker is trying to join the wrong cluster. Configured zookeeper.connect may be wrong.
  ```
How to fix?
- Delete Kafka stateful set `kubectl -n posthog delete sts posthog-posthog-kafka`

> - Delete Kafka pods `kubectl -n posthog delete pod posthog-posthog-kafka-0`

Reviewer: What happens to the Posthog instance while we run this?

Author: it goes down. It's similar to what happens with the fix for going from 1.x.x to 2.x.x

> - Delete old zk pod `kubectl -n posthog delete pod posthog-zookeeper-0`
> - Upgrade helm chart `helm upgrade -f values.yaml --timeout 20m --namespace posthog posthog posthog/posthog`

Reviewer: Based on what Jesse said this seems important part too.

```suggestion
- Delete old zk pod `kubectl -n posthog delete pod posthog-zookeeper-0`
- Wait for pods to spin down
- Upgrade helm chart `helm upgrade -f values.yaml --timeout 20m --namespace posthog posthog posthog/posthog`
```
